<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PatientListFrameWidget</class>
 <widget class="QWidget" name="PatientListFrameWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1193</width>
    <height>816</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(211, 211, 211);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>15</number>
   </property>
   <property name="leftMargin">
    <number>5</number>
   </property>
   <property name="topMargin">
    <number>10</number>
   </property>
   <property name="rightMargin">
    <number>5</number>
   </property>
   <property name="bottomMargin">
    <number>5</number>
   </property>
   <item>
    <widget class="QGroupBox" name="queryFilterGroupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>130</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QGroupBox {color: rgb(49, 79, 79);
border-width:2px;
border-style:solid;
	border-color: rgb(49, 79, 79);}
</string>
     </property>
     <property name="title">
      <string>查询条件</string>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>150</width>
        <height>35</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(49, 79, 79);</string>
      </property>
      <property name="text">
       <string>姓名</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>30</y>
        <width>150</width>
        <height>35</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(49, 79, 79);</string>
      </property>
      <property name="text">
       <string>检查号</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>800</x>
        <y>30</y>
        <width>150</width>
        <height>35</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(49, 79, 79);</string>
      </property>
      <property name="text">
       <string>报告状态</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="nameFilterEdit">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>30</y>
        <width>200</width>
        <height>35</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="accessionNumFilterEdit">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>30</y>
        <width>200</width>
        <height>35</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
      </property>
     </widget>
     <widget class="QComboBox" name="reportStatusFilterComboBox">
      <property name="geometry">
       <rect>
        <x>960</x>
        <y>30</y>
        <width>200</width>
        <height>35</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>150</width>
        <height>35</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(49, 79, 79);</string>
      </property>
      <property name="text">
       <string>检查时间</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QDateEdit" name="studyDateFromFilterDateEdit">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>80</y>
        <width>200</width>
        <height>35</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>80</y>
        <width>150</width>
        <height>35</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(49, 79, 79);</string>
      </property>
      <property name="text">
       <string>至</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QDateEdit" name="studyDateToFilterDateEdit">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>80</y>
        <width>200</width>
        <height>35</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
      </property>
     </widget>
     <widget class="ThemePushButton" name="queryButton">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>1010</x>
        <y>78</y>
        <width>150</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>查询</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="studyDateTodayFilterRadioButton">
      <property name="geometry">
       <rect>
        <x>750</x>
        <y>75</y>
        <width>80</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>本日</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="studyDateThisMonthFilterRadioButton">
      <property name="geometry">
       <rect>
        <x>750</x>
        <y>100</y>
        <width>80</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>本月</string>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QGroupBox {color: rgb(49, 79, 79);
border-width:2px;
border-style:solid;
	border-color: rgb(49, 79, 79);}
</string>
     </property>
     <property name="title">
      <string>检查列表</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>10</number>
      </property>
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="topMargin">
       <number>30</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <property name="bottomMargin">
       <number>10</number>
      </property>
      <item>
       <widget class="QWidget" name="widget" native="true">
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="spacing">
          <number>10</number>
         </property>
         <item>
          <widget class="ThemePushButton" name="sendPacsButton">
           <property name="font">
            <font>
             <pointsize>16</pointsize>
            </font>
           </property>
           <property name="toolTip">
            <string extracomment="选择检查列表的影像发送到PACS"/>
           </property>
           <property name="text">
            <string>PACS</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="ThemePushButton" name="importFilesButton">
           <property name="font">
            <font>
             <pointsize>16</pointsize>
            </font>
           </property>
           <property name="toolTip">
            <string extracomment="导入检查影像到工作站"/>
           </property>
           <property name="text">
            <string>导入影像</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="ThemePushButton" name="exportStudyButton">
           <property name="font">
            <font>
             <pointsize>16</pointsize>
            </font>
           </property>
           <property name="toolTip">
            <string extracomment="导出选中的检查影像到本地磁盘"/>
           </property>
           <property name="text">
            <string>导出检查</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="ThemePushButton" name="burnStudyButton">
           <property name="font">
            <font>
             <pointsize>16</pointsize>
            </font>
           </property>
           <property name="toolTip">
            <string extracomment="选中的检查影像刻录到光盘"/>
           </property>
           <property name="text">
            <string>光盘刻录</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="ThemePushButton" name="viewStudyButton">
           <property name="font">
            <font>
             <pointsize>16</pointsize>
            </font>
           </property>
           <property name="toolTip">
            <string extracomment="查看选中的检查影像"/>
           </property>
           <property name="text">
            <string>查看影像</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="ThemePushButton" name="viewReportButton">
           <property name="font">
            <font>
             <pointsize>16</pointsize>
            </font>
           </property>
           <property name="toolTip">
            <string extracomment="查看选中的检查报告"/>
           </property>
           <property name="text">
            <string>查看报告</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_2" native="true"/>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QTableView" name="patientTableView">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::Panel</enum>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::MultiSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <attribute name="horizontalHeaderVisible">
         <bool>true</bool>
        </attribute>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ThemePushButton</class>
   <extends>QPushButton</extends>
   <header>themepushbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
